shop/src/main/java/ro/msg/learning/shop/repository/StockRepository.java

```diff
@@ -0,0 +1,23 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.domain.Stock;
+import ro.msg.learning.shop.domain.key.StockId;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface StockRepository extends JpaRepository<Stock, StockId> {
+    List<Stock> findAllByProductProductId(UUID productId);
```

Comment:
Is this name right? Does the JpaRepository implement what you want behind the scenes?